modules/social_features/social_search/social_search.install

```diff
@@ -241,3 +222,17 @@ function social_search_update_10201() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Add the processor "Skip not Verified users" to indexes.
+ */
+function social_search_update_11001() {
```

Comment:
Doesn't this also require a full search reindex?

Reply:
I think no, and I'll try to explain why.
Since for existing users we have bulk assign "Verified" role then we won't have unverified users after it. And new unverified users won't be able to show on the search profiles page because of the filter.

Reply:
Thanks, that makes a lot of sense!